src/Parser/Core/Modules/Items/ArchimondesHatredReborn.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+class ArchimondesHatredReborn extends Module {
+  healing = 0;
+
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.owner.selectedCombatant.hasTrinket(ITEMS.ARCHIMONDES_HATRED_REBORN.id);
+    }
+  }
+
+  on_byPlayer_absorbed(event) {
+    const spellId = event.ability.guid;
+    if (spellId === ITEMS.ARCHIMONDES_HATRED_REBORN.absorbId) {
+      this.healing += event.amount;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.ARCHIMONDES_HATRED_REBORN,
+      result: (
+        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
```

Comment:
A bit nitpicky maybe, but I think this doesn't really give the user any more information so could be removed for the sake of keeping the amount of tooltips to a minimum so they actually read the ones that matter.